api/institutions/utils.py

```diff
@@ -0,0 +1,54 @@
+from rest_framework import exceptions
+from osf.models import Institution
+from osf.utils import permissions as osf_permissions
+from api.base.serializers import relationship_diff
```

Comment:
Nit-picking: since this is a new file, let's fix import order.

```python
from rest_framework import exceptions

from api.base.serializers import relationship_diff
from osf.models import Institution
from osf.utils import permissions as osf_permissions
```